src/views/Home.jsx

```diff
@@ -1,42 +1,31 @@
+import React, { useState } from 'react';
 import { SingleList } from '../components/SingleList';
 import './Home.css';
 import { createList } from '../api/firebase';
-import { useState } from 'react';
 import { Link, useNavigate } from 'react-router-dom';
 
-export function Home({ data, setListPath, userId, userEmail }) {
+export function Home({ data, listPath, setListPath, userId, userEmail }) {
 	const [name, setName] = useState('');
-	const [selectedList, setSelectedList] = useState(null); // State to track the selected list
 	const navigate = useNavigate();
 
 	const handleSubmit = async (event) => {
-		event.preventDefault(); // Prevent the default form submission behavior.
+		event.preventDefault();
 
 		try {
 			const newList = await createList(userId, userEmail, name);
-			setListPath(newList); // creates a new list and automatically creates the userId
-			// that tracks the purchased item and also saves it to local storage
-
-			setName(''); //refreshes the form after submission place takes it back to the default state
-
-			alert('The item has been added.'); //alert message
-
-			navigate('/list'); // navigate to list page
+			setListPath(newList);
+			setName('');
+			alert('The item has been added.');
+			navigate('/list');
 		} catch (err) {
 			console.error(err);
-
-			alert('item not created'); //alert message if there is an error with creating the item
+			alert('Item not created');
 		}
 	};
 
 	const handleChange = (e) => {
 		setName(e.target.value);
 	};
-	// to create a highlighted list
-	const handleListClick = (path) => {
-		setListPath(path);
-		setSelectedList(path); // this update and highlight the list path
-	};
```

Comment:
you should keep the `handleListCLick`, only remove the duplicate `setSelectedList(path)`